playbooks/post-config.yaml

```diff
@@ -43,3 +43,7 @@
         - zuul-merger
         - zuul-fingergw
         - zuul-web
+
+- name: Create Labsync task
+  import_playbook: sync-config-files.yaml
+  when: labsync_enabled
```

Comment:
you can use `when: labsync_enabled|default(false)|bool` here and remove all the `labsync_enabled: false`